package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -0,0 +1,587 @@
+    the `H5MD`_ file contains no units.
+
+    Currently reads .h5md files with the following HDF5 hierarchy:
+
+    .. code-block:: text
+
+        Notation:
+        (name) is an HDF5 group that the reader recognizes
+        {name} is an HDF5 group with arbitrary name
+        [variable] is an HDF5 dataset
+        <dtype> is dataset datatype
+        +-- is an attribute of a group or dataset
+
+        H5MD root
+         \-- (h5md)
+            +-- version <int>
+            \-- author
+                +-- name <str>, author's name
+                +-- email <str>, optional email address
+            \-- creator
+                +-- name <str>, file that created .h5md file
+                +-- version
+         \-- (particles)
+            \-- {group1}
+                \-- (box)
+                    +-- dimension : <int>, number of spatial dimensions
+                    +-- boundary : <str>, boundary conditions of unit cell
+                    \-- (edges)
+                        \-- [step] <int>, gives frame
+                        \-- [value] <float>, gives box dimensions
+                \-- (position)
+                    \-- [step] <int>, gives frame
+                    \-- [time] <float>, gives time
+                        +-- units <str>
+                    \-- [value] <float>, gives numpy arrary of positions
+                                         with shape (n_atoms, 3)
+                        +-- units <str>
+                \-- (velocity)
+                    \-- [step] <int>, gives frame
+                    \-- [time] <float>, gives time
+                        +-- units <str>
+                    \-- [value] <float>, gives numpy arrary of velocities
+                                         with shape (n_atoms, 3)
+                        +-- units <str>
+                \-- (force)
+                    \-- [step] <int>, gives frame
+                    \-- [time] <float>, gives time
+                        +-- units <str>
+                    \-- [value] <float>, gives numpy arrary of forces
+                                         with shape (n_atoms, 3)
+                        +-- units <str>
+         \-- (observables)
+            \-- (lambda)
+                \-- [step] <int>, gives frame
+                \-- [time] <float>, gives time
+                \-- [value] <float>
+            \-- (step)
+                \-- [step] <int>, gives frame
+                \-- [time] <float>, gives time
+                \-- [value] <int>, gives integration step
+
+    .. note::
+        The reader does not currently read mass or charge data.
+
+    .. note::
+        The ``_reopen()`` method does not explicity close and reopen
+        the file like most other coordinate readers, as this would
+        cause problems with reopening the file with the ``driver``
+        and ``comm`` arguments for parallel reads. Instead, it rewinds
+        the trajectory back to the first timstep.
```

Comment:
Copy from `_reopen()` and make sure that all texts agree with each other and the code.